roles/config-wireguard/config/sea.yml

```diff
@@ -94,3 +94,10 @@ wg_peers:
   wg_pubkey: /aY73VNAGQ7W+GersZUSO6PqHJV8nWKb12Op9EQzY3k=
   peer_v4: null
   peer_v6: fe80::207
+
+- name: dn42-dixon
+  port: 21329
+  remote: hades.andydixon.com:65500
+  wg_pubkey: DkAGcccVOPxTV3+1tqKdhh0JM3BFH/uEnQD27Dz93Ro=
+  peer_v4: 172.22.157.161
+  peer_v6: fdac:f7ec:efa7 
```

Comment:
You've specified an IPv6 next hop but no way to send IPv6 BGP routes. Usually this is done using a multiprotocol BGP session (adding the `ipv6` channel) in the BIRD config snippet, or by creating a separate IPv6 BGP session if  your BGP daemon doesn't support this.

Reply:
Also, Seattle is not the closest node to this remote - probably that would be Amsterdam, NL (node name `ams`)